sparkly/session.py

```diff
@@ -152,28 +170,6 @@ def builder(self):
             'Please, follow the documentation for more details.'
         )
 
-    def has_package(self, package_prefix):
```

Comment:
I am wondering if we should bump the major version in this case, because we may break code for the existing open-source users

Reply:
yeah, i thought about that too... @mantzouratos had the same thought. technically it is breaking. but i can't imagine the scenario where someone would be using this function?

Reply:
I agree that it's a weird case, but, technically, it might be used somewhere

Reply:
Yeah I agree with bumping the major. Wanted to commit a backwards-incompatible change at `testing` at some point (getting rid of some stuff we never use), so this sounds like a convenient coincidence

Reply:
updated to 3.0.0.dev1; when we want to cut the official we can drop the dev suffix